src/api/index.ts

```diff
@@ -84,7 +84,11 @@ export class Api {
   async fetchSummaryWithTimeseries(
     region: Region,
   ): Promise<RegionSummaryWithTimeseries | null> {
-    if (region instanceof State) {
+    if (region.fipsCode === '0') {
```

Comment:
since we're using this coded value in more than one place, consider using a named constant (e.g., USA_FIPS_CODE) instead of a string literal so 1) we can trace its usage better and 2) so it's clear what it represents.

Reply:
👍  This is a WIP.  Will clean up!